nin/backend/nin

```diff
@@ -63,6 +64,34 @@ program
     generate.generate(projectRoot, resource, name);
   });
 
+program
+  .command('update')
+  .description('Update nin to newest version')
+  .action(function() {
+    if(!which('nin')) {
+      echo('Can\'t find nin executable in PATH/PATHEXT');
+      echo('Make sure \'/path/to/nin/nin/backend\' is present.');
+      exit(0);
+    }
+    echo('Checking for new version of nin');
+
+    var ninExecutable = which('nin');
+    var ninPath = ninExecutable.split('/');
+    var ninPath = ninPath.splice(0, ninPath.length-3).join('/');
+
+    cd(ninPath);
+    var gitGoToMaster = exec('git checkout master');
+    if(gitGoToMaster.code == 1) {
+      echo('Could not update nin, please resolve conflicts before updating');
+      exit(0);
+    }
+
+    var gitUpdate = exec('git pull');
```

Comment:
Should check exit code. Changes in working space might make this command blow up.